src/components/views/SuggestedViews/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+import { Grid, makeStyles, Theme } from '@material-ui/core';
+
+import { getSuggestedViews } from 'utils/datasetUtils';
+import getViews from 'fetching/views/getViews';
+import { useUser } from 'hooks';
+import ViewCard from './ViewCard';
+import ZetkinQuery from 'components/ZetkinQuery';
+import ZetkinSection from 'components/ZetkinSection';
+import { ZetkinView } from 'types/zetkin';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+        [theme.breakpoints.up('sm')]: {
+            flexDirection: 'row',
+        },
+        marginBottom: theme.spacing(1),
+    },
+    item: {
+        flex: 1,
+    },
+}));
+
+const SuggestedViews: React.FunctionComponent = () => {
+    const classes = useStyles();
+    const router = useRouter();
+    const user = useUser();
```

Comment:
The owner of a view is not really a _user_ but a `Person`. The `Person` object is an individual from the perspective of an organization. It may or may not correspond to a Zetkin user (most people organized in Zetkin are not users of Zetkin, and many users of Zetkin actually have corresponding `Person` objects in more than one organization).

Comparing the ID of the current `User` with the ID of the view owner (`Person`) is thereby incorrect. It may work here, because the ID of the user and the ID of the corresponding person (_in this organization_) is the same in the dummy data. But it won't work in general.

Instead, one must retrieve the `Person` corresponding to the current `user`. There is a similar situation in `CampaignDetailsForm` which retrieves the membership and from that the `Person` (called `profile`). 

https://github.com/zetkin/app.zetkin.org/blob/16fc62754fcba8a438ac22280766c2a8a2bc7e3c/src/components/forms/CampaignDetailsForm.tsx#L25-L27

Maybe that can be broken out into a hook and reused here?

Reply:
Oups good point

Reply:
Am reusing this code for the sake of making the component pull the correct data. 
Hope you don't mind but not breaking out into a more abstract hook / provider for now - because I don't understand enough about the distinction between User and Person. If a User and Person are both present at the same time in at least some sessions on the app, then I feel like the UserContext might be refactored into something like "IdentityContext", but I don't have enough info to just go ahead and do that for now, and it in any case feels a bit like out of scope / should be a separate issue....